lib/doorkeeper/oauth/authorization/code.rb

```diff
@@ -15,8 +15,13 @@ def issue_token
           @token ||= AccessGrant.create! access_grant_attributes
         end
 
-        def native_redirect
-          { action: :show, code: token.plaintext_token }
+        def oob_redirect
+          case pre_auth.redirect_uri
```

Comment:
What is the reason to have `case` statement here? I don't see any diff in result, also I see `nil` for `else` branch.. It's scares me :)

Reply:
It is a just vestigial thinko on my part. I thought it was going be different but really it's the just the presentation/view can be slightly different. I remove it.

For the now the presentation remains the same for both.